packages/models/src/resource-binding/cell-change-buffer.ts

```diff
@@ -0,0 +1,56 @@
+import { Input } from '@ckb-lumos/base'
+import { ActorURI } from '../actor'
+import { ResourceBindingRegistry } from './interface'
+import { CellChangeData } from './types'
+
+export class CellChangeBuffer {
+  #buffer: Map<ActorURI, [ResourceBindingRegistry, Input[], CellChangeData[]][]> = new Map()
```

Comment:
`[ResourceBindingRegistry, Input[], CellChangeData[]]` is complex and reused in several places, it's better to assign it to a custom type.

Reply:
Seems that it's a `bufferMap` instead of a `buffer`?

I ask this because it is exported as `buffer` in https://github.com/ckb-js/kuai/pull/125/files#diff-4d312df7669a766201d2e72e4db825d5779742390b5e6290f189381535f02eb8R49-R51
And buffer is expected to be a data structure providing `push` and `pop`, but in this case the `buffer` doesn't have these methods.